project-base/storefront/components/Pages/CategoryDetail/AdvancedSeoCategories.tsx

```diff
@@ -6,17 +6,19 @@ type AdvancedSeoCategoriesProps = {
     readyCategorySeoMixLinks: CategoryDetailFragmentApi['readyCategorySeoMixLinks'];
 };
 
+const simpleNavigationItemTwClass = 'lg:justify-center text-center';
+
```

Comment:
I do not see many benefits of having this in a separate variable. I will leave this to you, but I would just use it as-is.

Reply:
The purpose of having this in separate variable is to be able to use IntelliSense to recognize the "TwClass" regex. We agreed on this with other FE guys. Maybe you missed this meeting, but this is only way currently to tell VS Code that this is meant for Tailwind classes. However you need to setup this regex on your side in your editor.